src/command_modules/azure-cli-billing/azure/cli/command_modules/billing/commands.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import cli_command
+from azure.cli.command_modules.billing._client_factory import \
+    (invoices_mgmt_client_factory,
+     billing_periods_mgmt_client_factory)
+from azure.cli.command_modules.billing._transformers import \
+    (transform_invoice_list_output,
+     transform_invoice_output,
+     transform_billing_period_output,
+     transform_billing_period_list_output)
+
+invoices_path = 'azure.mgmt.billing.operations.invoices_operations#{}'
+billing_periods_path = 'azure.mgmt.billing.operations.billing_periods_operations#{}'
+custom_path = 'azure.cli.command_modules.billing.custom#'
```

Comment:
Why do the others use the 'X#{}' format whilst this one is `custom_path + 'string'`?